### PR TITLE
fix(quantic): weird height issue in generated answers containing lists

### PR DESCRIPTION
**Pull request by SimonMilord**
[SFINT-5597](https://coveord.atlassian.net/browse/SFINT-5597)

## ISSUE:

- The generated answer generated had a `whitespace: 'pre-wrap'` CSS  style on it which was causing this extra height between elements of the generated lists.

## PROPOSED SOLUTION:
- Removing the `pre-wrap` I was suspicious about removing it since it was probably there for a reason. However, I did check with about 20 different queries and double checked with all sorts of screen widths and it seems to be working as expected.
- I also checked on atomic search pages in the builder, it did not seem to apply the pre-wrap.

## DEMO:

Before:
image

After:
image

https://github.com/user-attachments/assets/adb7a305-a2cd-4986-b87b-b4fbdcae4f30

Community link:
https://agility-data-1815-dev-ed.scratch.my.site.com/s/

[SFINT-5597]: https://coveord.atlassian.net/browse/SFINT-5597?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by mmitiche** [comment]
> Ps: Let me know if you need access to the scratch org to test this.

For easy testing, create a community page and make it public for guest users and just put the link of the community page in your PR, we will able to test without any credentials.

FYI @lvu285 , we will be able to test this issue in the community @SimonMilord  will share with us.

**Comment by SimonMilord** [comment]
> > Ps: Let me know if you need access to the scratch org to test this.
> 
> For easy testing, create a community page and make it public for guest users and just put the link of the community page in your PR, we will able to test without any credentials.
> 
> FYI @lvu285 , we will be able to test this issue in the community @SimonMilord will share with us.

added the link for the community, also shared creds with lucille and @mmitiche  via one password

**Review by mmitiche — APPROVED** [review]
LGTM!
If @lvu285  approves, it's all good on my side.

File: packages/quantic/force-app/main/default/lwc/quanticGeneratedAnswerContent/templates/generatedMarkdownContent.css [hunk 1]
```diff
@@ -100,6 +100,7 @@ blockquote {
 }
 
 .scrollable-table {
+  margin-top: var(--lwc-spacingLarge, 1.5rem);
```
Comment (SimonMilord) [inline packages/quantic/force-app/main/default/lwc/quanticGeneratedAnswerContent/templates/generatedMarkdownContent.css:103]: the removal of pre-wrap on white-space was causing a lack of margin on top of tables, so I fix it here. We were already setting margin on the bottom but not on the top.

before:

image


now:

image

**Comment by lvu285** [comment]
All looks good on my side. We are good to go ✔️

**Review by lvu285 — APPROVED** [review]

**Pull request merged**